helm/Chart.yaml

```diff
@@ -5,5 +5,7 @@ version: 0.1.0
 keywords:
   - Apache OpenWhisk
 maintainers:
+  - name: Dave Grove
```

Comment:
We should personal names and emails from ASF repos. H
How about:
Name: Apache OpenWhisk committers
email: dev@openwhisk.org